styles.css

```diff
@@ -0,0 +1,57 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  scroll-behavior: smooth;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background-image: url("media/background2.svg");
```

Comment:
- [ ] Good job using background image in `css` to add the style-ralated images 👏🏻